src/crypto/algorithms/megolm.js

```diff
@@ -656,6 +665,7 @@ MegolmDecryption.prototype.onRoomKeyEvent = function(event) {
     this._olmDevice.addInboundGroupSession(
         content.room_id, senderKey, sessionId,
         content.session_key, event.getKeysClaimed(),
+        exportFormat,
```

Comment:
Should we make the keysProved and keysClaimed dicts be empty until we have better idea of what the correct behaviour is here?

I'm slightly worried that the constraint on keys proved doesn't hold anymore. https://github.com/matrix-org/matrix-js-sdk/blob/v0.7.10/src/crypto/OlmDevice.js#L780-L782